client/gutenberg/editor/controller.js

```diff
@@ -142,6 +143,30 @@ export const redirect = ( { store: { getState } }, next ) => {
 	return page.redirect( `/post/${ getSelectedSiteSlug( state ) }` );
 };
 
+function getPressThisContent( query ) {
+	const { text, url, title, image, embed } = query;
+	const caption = !! text ? '' : '<figcaption><a href="${ url }">${ title }</a></figcaption>';
+	let content = '';
+
+	if ( isValidUrl( embed ) ) {
+		//TODO: this turns into an wp:html block
+		content = content + `<!-- wp:embed --><p>${ embed }</p><!-- /wp:embed -->`;
```

Comment:
The following doesn't work, but any better suggestions on how to programmatically add an embed from a url for initialEdits? cc @notnownikki @aduth

Reply:
You can probably use `rawHandler` from the `@wordpress/block` module, which is what's otherwise used to convert pasted text to an embed block:

```
wp.blocks.rawHandler( { HTML: 'https://twitter.com/CincinnatiZoo/status/1050770683536568322' } )
// [
//   {
//     "clientId": "7ca26d6e-05fe-43ca-a964-382157c85992",
//     "name": "core/embed",
//     "isValid": true,
//     "attributes": {
//       "url": "https://twitter.com/CincinnatiZoo/status/1050770683536568322",
//       "allowResponsive": true
//     },
//     "innerBlocks": []
//   }
// ]
```